docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1529,6 +1529,121 @@ the definition and the storage table.
 The connector supports {doc}`/admin/fault-tolerant-execution` of query
 processing. Read and write operations are both supported with any retry policy.
 
+## Table functions
+
+The connector provides the following table functions:
+
+### table_changes
+
+Allows reading row-level changes between two versions of an Iceberg table. 
+This is how these changes can be read:
+
+```sql
+SELECT
+  *
+FROM
+  TABLE(
+    system.table_changes(
+      schema_name => 'default',
+      table_name => 't1',
+      start_snapshot_id => 6541165659943306573,
+      end_snapshot_id => 6745790645714043599
+    )
+  );
+```
+
+`schema_name` - type `VARCHAR`, required, name of the schema for which the function is called
+
+`table_name` - type `VARCHAR`, required, name of the table for which the function is called
+
+`start_snapshot_id` - type `BIGINT`, required, the identifier of the exclusive starting snapshot
+
+`end_snapshot_id` - type `BIGINT`, required, the identifier of the inclusive end snapshot
+
+In addition to returning the columns present in the table, the function
+returns the following values for each change event:
+
+- `_change_type`
+  : Gives the type of change that occurred. Possible values are `insert` and `delete`.
+- `_change_version_id`
+  : Shows the identifier of the snapshot in which the table occurred.
+- `_change_timestamp`
+  : Represents the timestamp for the snapshot became active.
+- `_change_ordinal`
+  : Represents the order of the changes. Can be used for sorting the results.
+
+This is how the `table_changes` table function would be used:
+
+Create table:
+
+```
+CREATE TABLE test_schema.pages (page_url VARCHAR, domain VARCHAR, views INTEGER);
+```
+
+Insert data:
+
+```
+INSERT INTO test_schema.pages
+    VALUES
+        ('url1', 'domain1', 1),
+        ('url2', 'domain2', 2),
+        ('url3', 'domain1', 3);
```

Comment:
are the domain values here and below different on purpose?

Reply:
the `page_url` & `domain` are supposed to be the primary key so to say of the table that can be used in downstream processes to identify the history of changes performed on a tuple.

Note that the same example slightly adapted is employed in Trino Delta Lake connector as well.